noah/src/main/resources/kafka/kafka.conf

```diff
@@ -0,0 +1,46 @@
+#This is the configuration file for kafka
```

Comment:
just want to make sure that to use the Kafka we don't need this kafka config, right? 
it's only used to setup the kafka right?